docs/modules/user-guide/partials/proc_developing-with-devfiles.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc_developing-with-devfiles_{context}"]
+= Developing with devfiles
+[role="_abstract"]
+
+Use the devfile specification to develop a node.js “Hello World” application. Developing this application introduces you to how a devfile automates and simplifies your development process.
+
+.Prerequisites
+
+* To develop an application by using the devfile specification, you need to create a Kubernetes cluster on your local machine. To create such a cluster, install link:https://minikube.sigs.k8s.io/docs/start/[minikube].
+* To access your namespaces, download link:https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/[kubectl].
+* When you use minikube for the first time, you must enable Ingress to access your devfile projects:
+** Set up link:https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/[Ingress on minikube].
+** Enable Ingress by running `minikube addons enable ingress`.
+* To execute the devfile specification, install the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/cli_tools/developer-cli-odo#installing-odo[odo cli tool].
+
+.Procedure
+
+. Run `minikube start` to build your Kubernetes cluster.
```

Comment:
The first two points seem redundant. Minikube installation is not described, and yet we now go into the 'minikube start' kind of detail.